src/Servers/Kestrel/Core/src/Internal/Http3/Http3Connection.cs

```diff
@@ -923,6 +925,11 @@ internal WebTransportSession OpenNewWebTransportSession(Http3Stream http3Stream)
         return session;
     }
 
+    private Http3ErrorCode GetErrorCodeOrNoError()
+    {
+        return _errorCodeFeature.Error <= 0 ? Http3ErrorCode.NoError : (Http3ErrorCode)_errorCodeFeature.Error;
```

Comment:
Will it be something other than -1 by default? I'd special case that value as getting defaulted to `Http3ErrorCode.NoError`. If someone explicitly sets an invalid value using the feature then they can get an error.
```suggestion
        // The default error value is -1. If it hasn't been changed before abort is called then default to HTTP/3's NoError value.
        return _errorCodeFeature.Error == -1 ? Http3ErrorCode.NoError : (Http3ErrorCode)_errorCodeFeature.Error;
```

Reply:
I started with a comparison to `-1`, but I changed it because the other consumers check for `<= 0`.  I'd rather not update them this late though, so I guess we'll just live with the inconsistency in RC2.